apps/hip/templates/includes/search_result_card.html

```diff
@@ -1,19 +1,30 @@
-<div class="card my-4">
+{% load wagtailcore_tags %}
+
+<div class="card my-4 search-page-result-card-hip">
   <div class="card-content">
     <div class="content">
-      <div>
-        <a href="#a-result-page"><strong>COVID-19 Vaccine Availability chart</strong></a>
-      </div>
-      <div class="py-2">
-        {# I think this is a pdf of the webpage above. But this is something we should check with Erik#}
-        <h6 class="is-size-6">hip.phila.gov/Resources/COVID-19-vaccine/pdf</h6>
-      </div>
-      <div>
-        {# placeholder image, presumably showing a thumbnail or image of a page result #}
-        <figure class="image is-128x128 px-0 mx-0">
-          <img src="https://bulma.io/images/placeholders/128x128.png">
-        </figure>
+
+      <p><a href="{% pageurl result %}"><strong>{{ result }}</strong></a></p>
+
+      <div class="level">
+        <div class="level-left">
+          <div class="level-item">
+            <p class="search-page-url-hip">{{ result.full_url }}</p>
+          </div>
+        </div>
+        <div class="level-right">
+          {% if result.specific.get_priority_display %}
+            <div class="level-item search-page-priority-badge-hip {{ result.specific.get_priority_display|slugify }}-hip">
+              <i class="fa {{ result.specific.get_priority_icon }}"></i>
+              {{ result.specific.get_priority_display|upper }}
+            </div>
+          {% endif %}
+        </div>
       </div>
+
+      {% if result.search_description %}
+        <p>{{ result.search_description|safe }}</p>
```

Comment:
Should this field have any values? After I ran the `update_index` command locally, this field is still blank, so nothing shows up here for any of the cards.

Reply:
Oops, I meant to mention this in the PR description, but forgot. The [`search_description`](https://docs.wagtail.io/en/stable/reference/pages/model_reference.html#wagtail.core.models.Page.search_description) field is available in the "Promote" tab for all pages. Although, I believe the documentation there is incorrect. It does NOT seem to be indexed.

My feeling is that we should recommend that editors always update the search_description field with the content that they want to see in the search result card. We might be able to do a fallback where if search_description is not present we look for another field (like 'description'?) but not all models will have that field. The question I have is whether we should INDEX the search_description field. It is not currently indexed, which makes sense to me since it is not shown on the page, and I don't like the idea of a page showing up in results for a term, but that term not being present on the page. But I suppose we could put it in the `<meta name="description">` field... thoughts?